Sources/RingPublishingTracking/Private/EventsFactory.swift

```diff
@@ -84,6 +83,9 @@ final class EventsFactory {
 
         if let contentMetadata = contentMetadata {
             parameters["DX"] = contentMetadata.dxParameter
+
+            let contentMarkAsPaidParameters = ContentMarkAsPaidDecorator(contentMetadata: contentMetadata).parameters
+            parameters["RDLCN"] = contentMarkAsPaidParameters["RDLCN"]
```

Comment:
jak teraz patrze to myśle głosno, że lepiej by to mieć jako rozszerzenie tak samo jak dxParameter niż dekorator